src/commands/misc/dog.js

```diff
@@ -18,9 +18,13 @@ module.exports = class Dog extends Command {
         message.channel.send({embed})
       } else {
         request.get('https://random.dog/woof.json').then(r => {
-          const embed = this.client.getDefaultEmbed(message.author)
-          embed.setImage(r.body.url).setDescription('Here is your dog <:DoggoF:445701839564963840>')
-          message.channel.send({embed})
+          if (r.body.url.endsWith(notsupported)) {
+            request.get('https://random.dog/woof.json').then(r => {

```

Comment:
The problem wasn't the if, in the second request it just can get another .mp4 and will return another blank embed

Reply:
lmao, 3 times once, i tried like, 100 times to request and, thats impossible

Reply:
The use of recursive function in this case prevents repeated code and the possibillity of requesting three MP4 in a row.